games-engines/gemrb/gemrb-0.8.5.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="An infinite game engine generator"
+DESCRIPTION="GemRB (Game Engine Made with preRendered Background) \
+is a port (actually a new implementation) of the original Infinity \
+Engine (the one of Baldur's Gate, Icewind Dale, Planescape: Torment, ...) \
+to Linux/Unix, MacOS X and Windows with some enhancements. Would \
+you like to create a game like Baldur's Gate?"
+HOMEPAGE="https://github.com/gemrb/gemrb"
+COPYRIGHT="2003-2017 The GemRB Team"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/gemrb/gemrb/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="d927e1eebd13a4643db363087fc61c5d2242e8a614da261f15bf4e2b1d94965c"
+PATCHES="gemrb-$portVersion.patchset"
```

Comment:
Oops, you forgot to insert the `SOURCE_FILENAME`:
```diff
+SOURCE_FILENAME="gemrb-$portVersion.tar.gz"
 PATCHES="gemrb-$portVersion.patchset"
```
According to [wiki/HaikuPorter-BuildRecipes#source_filename](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#source_filename), this is a possibility, but there's another page which says that a recipe should define a `SOURCE_FILENAME` in order to rename the tarball if the original name does not include the basename of the package (or at least something close enough to it).
(Sorry, I can't find that page right now...)